src/lib/base-command.js

```diff
@@ -212,17 +225,39 @@ module.exports = {
     }),
     target: Flags.string({
       char: 's',
-      description:
-        "The target instance type; one of 'author' or 'publish'. If not specified, deployments target both 'author' and 'publish' instances.",
+      description: "The target instance type. Default 'author'.",
```

Comment:
@jsedding the default has changed due to the logging command. is this an isssue at all?